Worker.Extensions.Sql/src/packages.lock.json

```diff
@@ -238,6 +243,15 @@
           "System.Threading.Tasks": "4.3.0"
         }
       },
+      "System.IO.FileSystem.AccessControl": {
```

Comment:
Hm, so this whole versioning thing makes me wonder if we have to also be checking new dependencies...because if we're adding new references like this that don't match what is currently in the bundle couldn't that also cause problems?

@lucyzhang929 @MaddyDev Could you follow up with checking this with the AF Host team?

Reply:
Will reach out to the AF host team.